statescript/executor.go

```diff
@@ -165,6 +167,19 @@ func execute(name string, timeout time.Duration) int {
 		return retCode(err)
 	}
 
+	bts, err := ioutil.ReadAll(stderr)
+	if err != nil {
+		log.Error(err)
+	}
+
+	if len(bts) > 0 {
+		if len(bts) > 10*1024*8 { // do not log more than 10KB at a time
+			log.Errorf("More than 10KB collected from stderr of %s, data will not be written to log file", name)
```

Comment:
How about logging warning and truncating the message to 10kB, so that we are storing something always?